package.json

```diff
@@ -12,11 +12,14 @@
   "author": "Sabeel Khan",
   "license": "ISC",
   "dependencies": {
+    "cloudinary": "^1.39.0",
     "dotenv": "^16.3.1",
     "ejs": "^3.1.9",
     "ejs-mate": "^4.0.0",
     "express": "^4.18.2",
-    "mongoose": "^7.3.4"
+    "mongoose": "^7.4.1",
```

Comment:
Why are we upgrading mongoose here use `npm i mongoose@7.3.4` to install specific version